esmvaltool/recipes/ref/recipe_REF_CRE.yml

```
@@ -0,0 +1,101 @@
+# ESMValTool
+---
+documentation:
+  title: Model evaluation with focus on the cloud radiative effect.
+  description: >
+    Plot climatologies and zonal mean profiles of cloud radiative effects (sw + lw) for the REF.
+  authors:
+    - bock_lisa
+    - lauer_axel
+  maintainer:
+    - bock_lisa
+
+
+# Note: the following models are just examples
+datasets:
+  - {project: CMIP6, dataset: MPI-ESM1-2-LR, exp: historical, ensemble: r1i1p1f1, grid: gn}
+
+timerange_for_models: &time_period
+  timerange: '2005/2014'  # can be specified, this is just an example
+
+
+preprocessors:
+
+  full_climatology:
+    climate_statistics:
+      period: full
+    regrid:
+      target_grid: 1x1
+      scheme:
+        reference: esmf_regrid.schemes:ESMFAreaWeighted
+
+  zonal_mean:
+    climate_statistics:
+      period: full
+    regrid:
+      scheme:
+        reference: esmf_regrid.schemes:ESMFAreaWeighted
+      target_grid: 1x1
+    zonal_statistics:
+      operator: mean
+
+
+diagnostics:
+
+  plot_lwcre_maps:
```

Maybe simply "plot_maps" to reduce potential confusion?

```suggestion
  plot_maps:
```